authority/authority.go

```diff
@@ -118,6 +133,12 @@ func (a *Authority) init() error {
 	return nil
 }
 
+// GetDatabase returns the authority database. It will return null if it's not
```

Comment:
The comment is "sort of" correct. Generally speaking this method will return the SimpleDB (since that is the default if nothing is configured). But if you create an authority w/out running `.Init`, then you'll get nil.

Reply:
@dopey Does it makes sense to allow the switch from SimpleDB to Badger or any other type?

Reply:
I believe that we've decided that we won't support switching databases for reload.

Reply:
I think you should just say "GetDatabase returns the authority database.", without the rest. It won't return nil, if the DB is not configured.